### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by garageeks** [RudolphRiedel/FT800-FT813#27] [General]
Dear Rudolph,
I managed to get the Arduino example up and running on PlatformIO using a Riverdi display, that you may add to the tested list
Screen model RiTFT-70 part RVT70AQBNWN00  
https://riverdi.com/product/ritft70/

With the source code, I could learn how to use the library by trial and error. It is easier to comprehend compared to the Riverdi example, which is using the textbook Bridgetek code, with a lots of low level GPU commands.
However, results would be achieved quicker with some documentation. Are the commands your invention or is there a documentation i.e. from Bridgetek, on which the commands are based on?

Thank you for your good work!
Nick

**Comment by RudolphRiedel** [comment]
The RVT70UQFNWC0x already is on the list and it uses the exact same timings.

The functions follow the documentation in Bridgetek's programming guide as close as possible.

So for example we have CMD_BUTTON documented as:
void cmd_button( int16_t x, int16_t y, int16_t w, int16_t h, int16_t font, uint16_t options, const char* s );
And my library has this:
void EVE_cmd_button(int16_t x0, int16_t y0, int16_t w0, int16_t h0, int16_t font, uint16_t options, const char* text)

So except for the "EVE_" prefix and small variations in the names of the parameters this is following the documented API.

Now I also have this:
void EVE_cmd_button_burst(int16_t x0, int16_t y0, int16_t w0, int16_t h0, int16_t font, uint16_t options, const char* text)

But apart that these ..._burst() functions are to be used within an EVE_start_cmd_burst();/EVE_end_cmd_burst(); pair and speed things up by not transferring the start address, the API is the same.

The display list commands are a bit different though:
dl( VERTEX2F(100 * 16,62 * 16) ); -> EVE_cmd_dl(VERTEX2F(100 * 16,62 * 16));

But I just noticed that the programming guide is not consistent in the examples:
dl( TAG(1) );
cmd(TAG(33));

This is even on the same page...

So that would be:
EVE_cmd_dl(TAG(1)); or EVE_cmd_dl_burst(TAG(1));
